src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import PlainButton from '../../../../components/plain-button';
+import ArrowArcRight from '../../../../icon/fill/ArrowArcRight';
+
+import type { GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+
+import CommentForm from '../comment-form';
+
+import messages from './messages';
+import './ActivityThreadReplyForm.scss';
+
+type Props = {
+    currentUser?: User,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
+    mentionSelectorContacts?: SelectorItems<>,
+    onReplyCreate: Function,
```

Comment:
Please change the type here to arrow function that specifies expected attributes and their types (the same way it's defined in `ActivityThread`). This also applies to `getMentionWithQuery` above.

Reply:
I wasn't able to find how should `getMentionWithQuery` be typed though.